python/setup.py

```diff
@@ -43,7 +43,8 @@
     'scikit-learn',
     'packaging',
     'interpret-core[required]==0.1.19',
-    'shap>=0.20.0, <=0.33.0'
+    'shap>=0.20.0, <=0.33.0',
+    'flask'
```

Comment:
oh no, we don't want to depend on flask and all of its dependencies... hmm...

Reply:
discussed with @rihorn2 , we can add an extra here that will include all of the visualization dependencies similar to interpret-core/interpret